api/v1alpha1/sleepinfo_types.go

```diff
@@ -29,17 +29,27 @@ type ExcludeRef struct {
 
 // SleepInfoSpec defines the desired state of SleepInfo
 type SleepInfoSpec struct {
+	// WeekdaySleep are in cron notation.
+	//
+	// For example, to configure a schedule from monday to friday, set it to "1-5"
+	//+operator-sdk:csv:customresourcedefinitions:type=spec
+	WeekDaySleep string `json:"weekdaySleep,omitempty"`
+	// WeekDayWakeUp are in cron notation.
+	//
+	// For example, to configure a schedule from monday to friday, set it to "1-5"
+	//+operator-sdk:csv:customresourcedefinitions:type=spec
+	WeekDayWakeUp string `json:"weekdayWakeUp,omitempty"`
 	// Weekdays are in cron notation.
 	//
 	// For example, to configure a schedule from monday to friday, set it to "1-5"
 	//+operator-sdk:csv:customresourcedefinitions:type=spec
-	Weekdays string `json:"weekdays"`
+	Weekdays string `json:"weekdays,omitempty"`
```

Comment:
We could set this field as deprecated, since it should not be used in favor of the new fields. What do you think? Maybe using the markers: https://book.kubebuilder.io/reference/markers/crd